drop-in.php

```diff
@@ -0,0 +1,95 @@
+<?php
```

Comment:
Do we need this at all for testing, now that we have the `jetpack_photon_noresize_mode` filter?